examples/helpers/mail/MultipleEmailsMultipleRecipients.java

```diff
@@ -0,0 +1,46 @@
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import com.sendgrid.helpers.mail.objects.Personalization;
+
+import java.io.IOException;
+
+public class MultipleEmailsMultipleRecipients {
+
+  public static void main(String[] args) throws IOException {
+    final Mail mail = new Mail();
+
+    mail.setFrom(new Email("test@example.com", "Example User"));
+    mail.setSubject("Sending with Twilio SendGrid is Fun");
+
+    final Personalization personalization1 = new Personalization();
+    personalization1.addTo(new Email("test1@example.com", "Example User1"));
+    mail.addPersonalization(personalization1);
+
+    final Personalization personalization2 = new Personalization();
+    personalization2.addTo(new Email("test2@example.com", "Example User2"));
+    mail.addPersonalization(personalization2);
+
+    mail.addContent(new Content("text/plain", "and easy to do anywhere, even with Java"));
```

Comment:
I think for this example, folks would want to see how to include different messages for each recipient as well. The two ways to do that are with `substitutions` and `transactional templates` ([Examples in C#](https://github.com/sendgrid/sendgrid-csharp/blob/master/USE_CASES.md#send-multiple-emails-to-multiple-recipients)).

Reply:
Updated.